rclcpp/include/rclcpp/node_impl.hpp

```diff
@@ -250,6 +250,22 @@ Node::get_parameter_or(
   return got_parameter;
 }
 
+template<typename ParameterT>
+void
+Node::get_parameter_or_set_default(
```

Comment:
I think this could be shortened to `get_parameter_or_set`, but that's subjective of course.

Reply:
> I think this could be shortened to `get_parameter_or_set`, but that's subjective of course.

True.  I don't have a strong opinion, though I typically prefer a bit more verbose.  I'm going to leave it as `get_parameter_or_set_default` unless someone voices a strong opinion for something else.

(and thanks for the review!)

Reply:
Since it isn't setting the default value (since a parameter in the server side doesn't have a default value I would second the suggestion from @sloretz.

Reply:
All right, I renamed to `get_parameter_or_set` in  0bd546d.